### ISSUE TITLE
Messaging - Move Kedro advantages to website 

### ISSUE THREAD

**Issue by NeroOkwa** [kedro-org/kedro-website#72]
## Context
This involves moving the ‘[What are the primary advantages of Kedro?](https://kedro.readthedocs.io/en/stable/faq/faq.html)’ section from the FAQ on [kedro docs](https://kedro.readthedocs.io/en/stable/faq/faq.html) to the [kedro website](http://kedro.org/).

![image](https://user-images.githubusercontent.com/50266242/205914203-0b6de55b-b1d4-41b8-8f4b-1bc816e6ae12.png)

![image](https://user-images.githubusercontent.com/50266242/205914370-8e605225-6682-452a-9f6a-d4a3744a2c7f.png)

## Why is this important? 
This would clearly highlight the value proposition of Kedro  to Data Scientists, Machine-Learning Engineers or Data Engineers, and Project Leads, increasing awareness and adoption. 

## Next Steps

- [ ] Move content from FAQ to website page  
- [ ] Review content and web page design
- [ ] Release

**Comment by tynandebold** [comment]
We'll need to make a call here on how we present this information for different persona audiences. The website presents now mostly to the engineering persona, and we already do list the advantages of using Kedro there.

**Comment by NeroOkwa** [comment]
@tynandebold  Agreed! I think the FAQ section above targeted at Data Practitioners, MLE Engineers, and Project Leads is a good starting point, but a next level could be specifying advantages for both engineering and non-engineering personas.
